docs/source/docs/command-reference/context_free/transform.md

```diff
@@ -498,6 +499,31 @@ Examples:
     --id '2010_001705:train' --attr 'occluded'
   ```
 
+#### `astype_annotations`
+
+Enables the conversion of annotation types for the categories and individual items within a dataset.
+This transform only supports tabular datasets. If you want to change annotation types in datasets of other types, please use a different transform.
+
+Based on default setting it transforms the annotation types, changing them to 'Label' if they are categorical,
+and to 'Caption' if they are of type string, float, or integer.
+If you specifically set mapping, change annotation types based on the mapping.
+
+Usage:
+```console
+astype_annotations [-h] [--mapping MAPPING]
+```
+
+Optional arguments:
+- `-h`, `--help` (flag) - Show this help message and exit
+- `--mapping` (str) - Annotations type in the form of: '<src>:<dst>' (repeatable)
+
+Examples:
+- Convert type of `title` and `rating` annotation
+  ```console
+  datum transform -t astype_annotations -- \
+    --mapping 'title:text,rating:int'
```

Comment:
Lastly, have you updated the int to label?

Reply:
Fix in [fce21c5](https://github.com/openvinotoolkit/datumaro/pull/1484/commits/fce21c555af3403a92da981a80e517cb3cbdc67c)